force-app/main/adapter/in/sobjects/contact/Contacts.cls

```diff
@@ -336,4 +336,14 @@ public inherited sharing class Contacts extends fflib_SObjects {
        return getIdFieldValues(Contact.Id);
     }
 
+    public Contacts contactsWithNonNullAccountIds() {
+        List<Contact> contacts = new List<Contact>();
+        for (Contact c :(List<Contact>)  getRecords()) {
```

Comment:
Can we replace the variable `c` with something more descriptive, like `contactWithAccount`